src/SignalR.Orleans/Clients/ClientGrain.cs

```diff
@@ -8,107 +7,108 @@
 using Orleans.Streams;
 using Orleans.Concurrency;
 using SignalR.Orleans.Core;
+using System.Diagnostics;
 
 namespace SignalR.Orleans.Clients
 {
-    [DebuggerDisplay("{DebuggerDisplay,nq}")]
-    internal class ClientState
-    {
-        private string DebuggerDisplay => $"ServerId: '{ServerId}'";
-
-        public Guid ServerId { get; set; }
-    }
-
-    [Reentrant]
-    internal class ClientGrain : Grain, IClientGrain
+    /// <inheritdoc cref="IClientGrain"/>
+    internal sealed class ClientGrain : Grain, IClientGrain
```

Comment:
We should convert all those grains to POCO grains. Use `IGrainBase` instead of inherithing from `Grain`.